USER_GUIDE.md

```diff
@@ -478,9 +485,11 @@ print(response)
 
 Make sure to use `AsyncOpenSearch` with the `AsyncHttpConnection` connection class with the async `AWSV4SignerAsyncAuth` signer.
 
-```bash
-pip install opensearch-py[async]
-```
+#### Pre-requisites to use `AWSV4SignerAsyncAuth` 
+- Setup the async client as given [here](https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md#setup)
```

Comment:
I would just say, "Requires `opensearch-py[async]`.", keep it short. I don't think this needs a new section.